docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -121,7 +121,7 @@ type Asset = {
   * `launchAsset`: A special asset that is the entry point of the application code. The `fileExtension` field will be ignored for this asset and SHOULD be omitted.
   * `assets`: An array of assets used by the update bundle, such as JavaScript, pictures, and fonts. All assets (including the `launchAsset`) should be downloaded to disk before executing the the update, and a mapping of asset `key`s to locations on disk should be provided to application code.
   * Properties of each asset object:
-    * `hash`: SHA-256 hash of the file to guarantee integrity.
+    * `hash`: Hex-encoded SHA-256 hash of the file to guarantee integrity.
```

Comment:
```suggestion
    * `hash`: Case-insensitive, hex-encoded SHA-256 hash of the file to guarantee integrity.
```

(or specify casing if that is important -- either way, I think it is helpful to answer the question of case sensitivity here)